src/In.ProjectEKA.DefaultHip/In.ProjectEKA.DefaultHip.csproj

```diff
@@ -14,5 +14,9 @@
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.0" />
     <PackageReference Include="NpgSql.EntityFrameworkCore.PostgreSQL.Design" Version="1.1.1" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <ProjectReference Include="..\In.ProjectEKA.OtpService\In.ProjectEKA.OtpService.csproj" />
+  </ItemGroup>

```

Comment:
Why OtpService is referenced in DefaultHip?

Reply:
resolved